### PR TITLE
update README with usage and FAQ

### PR DESCRIPTION
**Pull request by pk1130**
Reference Issue #10: Task 3 involves updating the docs with usage and frequently-asked questions. I created a preliminary version of the same. Please review and add suggestions/feedback/queries that I can incorporate into the documentation! Thanks a ton!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,21 +23,55 @@ py-irt is now available on PyPi!
...
+## FAQ
+
+1. What kind of output should I expect on running the command to train an IRT model?
+
+You should see something like this when you run the command given above:
+![image](https://user-images.githubusercontent.com/40918514/123986740-3eccd100-d9cf-11eb-8e58-ba5ad6c977ce.png)
+
+2. I tried installing py-irt using pip from PyPI. But when I try to run the command `py-irt train 4pl ~/path/to/dataset/eg/squad.jsonlines /path/to/output/eg/test-4pl/`, I get an error that says `bash: py-irt: command not found`. How do I fix this?
+
+You may be running into this error because of the old version on PyPI, which has not yet been configured for CLI usage. It's in the pipeline and you should be able to see it soon! In the meantime, do the following steps for replicating CLI function in the repo using these commands:
```
Comment (jplalor) [inline README.md:57]: Can you update this to say: "The CLI interface was implemented in PyPi version 0.2.1. If you are getting this error try updating py-irt:

```shell
pip install --upgrade py-irt
```

Alternatively, you can install the latest version from github: (include shell instructions from the below)

File: README.md [hunk 2]
```diff
@@ -22,21 +23,55 @@ py-irt is now available on PyPi!
...
+```shell
+git clone https://github.com/nd-ball/py-irt.git
+cd py-irt
+mv ~/py-irt/py_irt/cli.py ~/py-irt/
+python cli.py train 4pl ~/path/to/dataset/eg/squad.jsonlines /path/to/output/eg/test-4pl/
+```
+
+3. How do I evaluate a trained IRT model?
+
+Evaluation of a trained IRT model is yet to be implemented. You're welcome to implement it and open a pull request!
```
Comment (jplalor) [inline README.md:68]: "Model evaluation is coming soon"

(I'll be implementing this next week, so no need for others to duplicate effort :) )

**Review by jplalor — CHANGES_REQUESTED** [review]
Thanks for these updates! I made a few comments on the text, could please update?

File: README.md (hunk 1 above)
Comment (pk1130) [reply README.md]: Updated!

File: README.md (hunk 2 above)
Comment (pk1130) [reply README.md]: Updated!

**Comment by pk1130** [comment]
Thanks for the quick review and suggestions! Made the necessary changes :) Think this is ready for merging :)

**Review by pk1130 — COMMENTED** [review]
All changes requested on commit 1 have been made in commit 2. Let me know if there are any other changes to be made :)

**Review by jplalor — APPROVED** [review]
Thanks!

**Pull request merged**